src/templates/next/components/Image/Image.tsx

```diff
@@ -0,0 +1,27 @@
+import { ImageProps } from "~/common"
+
+const Image = ({ src, alt, width }: ImageProps) => (
+  <img src={src} alt={alt} width={`${width ?? 100}%`} />
```

Comment:
curious why we added `height=auto` for classic but not here?

Reply:
tiptap put as auto, so i just added it in. 

dont have strong opinions about this at the moment, will just add this to the classic as well